mxabierto/adela
# feature-enviroment-variables-footer-and-fix-css

### Changelog

* Adding footer from cdn and correct styles necessaries, the next files has been changed:
* M: app/assets/stylesheets/application.css.scss
* M: app/views/admin/base/index.html.haml
* M: app/views/inventories/datasets/index.html.haml
* M: app/views/layouts/application.html.haml
* M: config/initializers/cdn_footer.rb
### How to test
* TODO: Follow the next steps:
1) Starting up your local environment.
2) Create an environment variable  called CDN_URL in you system with: 
   export CDN_URL='https://cdn.datos.gob.mx/qa/bower_components/'

You must see:
* https://cdn.datos.gob.mx/qa/bower_components/polymer/polymer.html 
* https://cdn.datos.gob.mx/qa/bower_components/dgm-footer/dgm-footer.html 
* https://cdn.datos.gob.mx/qa/bower_components/dgm-style/reset-footer.css 

3) Set the value https://cdn.datos.gob.mx/bower_components/ to thevariable CDN_URL                      
   with the next line: 
   export CDN_URL='https://cdn.datos.gob.mx/bower_components/'

You must see:
* https://cdn.datos.gob.mx/bower_components/polymer/polymer.html 
* https://cdn.datos.gob.mx/bower_components/dgm-footer/dgm-footer.html 
* https://cdn.datos.gob.mx/bower_components/dgm-style/reset-footer.css
